src/manager/index.js

```diff
@@ -9,6 +9,7 @@ export function register () {
             title: 'Locales',
             render: (props) => (
                 <LocalePanel
+                    key={props && typeof props === 'object' && props.key || undefined}
```

Comment:
What do you think about removing the first check `props`? Because it's already covered by the `typeof` check.

Reply:
@floriangosse It's a short circuit to avoid the ugly fact that `typeof null` is `object`

Reply:
Good point, hadn't this case in mind.